File: 백준/Bronze/1546. 평균/평균.py
count = int(input())
number = list(map(int, input().split()))

max_value = max(number)

for i, num in enumerate(number):
    number[i] = (num / max_value) * 100


sum_value = sum(number)
avg_value = sum_value / len(number)

print(avg_value)

File: 백준/Bronze/1546. 평균/평균.js
let fs = require("fs");
let input = fs.readFileSync("/dev/stdin").toString().split("\n");

let n = Number(input[0]);
let score = input[1].split(" ").map(Number);


let max = Math.max(...score);

score = score.map(num => (num / max) * 100);

let sum = 0;

for(let i = 0; i < score.length; i++){
  sum += score[i];
}

console.log((sum / n).toFixed(2))